AdyenCard/Utilities/BinLookup/BinLookupRequest.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright (c) 2020 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+internal struct BinLookupRequest: Request {
+    
+    internal typealias ResponseType = BinLookupResponse
+    
+    internal var path: String { "checkoutshopper/v1/bin/binLookup?token=\(clientKey)" }
+    
+    internal var counter: UInt = 0
+    
+    internal var headers: [String: String] = [:]
+    
+    internal var queryParameters: [URLQueryItem] = []
+    
+    internal var method: HTTPMethod = .post
+    
+    internal var clientKey: String
+    internal var encryptedBin: String
+    internal var supportedBrands: [CardType]
```

Comment:
if you just make `CardType` `Encodable`, then you don't need to implement `func encode(to encoder: Encoder)`, Swift compiler does the implementation for you.

Reply:
👍